samza-core/src/main/java/org/apache/samza/clustermanager/ClusterBasedJobCoordinator.java

```diff
@@ -178,9 +185,33 @@
   public ClusterBasedJobCoordinator(Config coordinatorSystemConfig) {
```

Comment:
Should we rename this to something more general now that it won't always be just the coordinator system configs?

Reply:
I was planning to rename this variable when cleaning up the legacy flow, but we can choose a generic name in the mid now as well, what do you think?

Reply:
I think changing it now would be less misleading in the interim. "jobCoordinatorConfig"?

Reply:
Renamed to jobCoordinatorConfig and updated the javadoc as well.